io_scene_nif/modules/nif_import/property/nodes_wrapper/__init__.py

```diff
@@ -134,6 +134,10 @@ def connect_to_pass(self, b_node_pass, b_texture_node, texture_type="Detail"):
             if texture_type in ("Detail", "Reflect"):
                 rgb_mixer.inputs[0].default_value = 1
                 rgb_mixer.blend_type = "OVERLAY"
+            # these textures are multiplied with the base texture (currently only vertex color)
+            elif texture_type == "Vertex_Color":
```

Comment:
Do we need to map this over on the export side or this ok for now?

Reply:
I think it is ok for now. As far as I can tell, the blending mode of the mixing node has no influence on how the vertex colors get exported, and that's the only thing I changed.

Reply:
Wondering if this is what we can use for APPLY_MODE mapping